src/gui/mzroll/forms/settingsform.ui

```diff
@@ -425,53 +396,259 @@
+                 <property name="toolTip">
+                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Smoothness (&lt;span style=&quot; font-weight:600;&quot;&gt;λ&lt;/span&gt;) for flexibility of the baseline. This value is represented as a linear scale for log of &lt;span style=&quot; font-weight:600;&quot;&gt;λ&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 </property>
+                 <property name="text">
+                  <string>Smoothness (λ)</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="0">
+                <widget class="QLabel" name="asymmetryLabel">
+                 <property name="toolTip">
+                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Asymmetry (&lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt;) for position bias of the baseline. The value of &lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt; needs to be in the range (0.00 ≤ &lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt; ≤ 0.1) to perform well on positive peak chromatograms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 </property>
+                 <property name="text">
+                  <string>Asymmetry (p)</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="2">
+                <widget class="QLabel" name="asymmetryValue">
+                 <property name="toolTip">
+                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Asymmetry (&lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt;) for position bias of the baseline. The value of &lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt; needs to be in the range (0.00 ≤ &lt;span style=&quot; font-weight:600;&quot;&gt;p&lt;/span&gt; ≤ 0.1) to perform well on positive peak chromatograms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 </property>
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="0" column="2">
+                <widget class="QLabel" name="smoothnessValue">
+                 <property name="toolTip">
+                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Smoothness (&lt;span style=&quot; font-weight:600;&quot;&gt;λ&lt;/span&gt;) for flexibility of the baseline. This value is represented as a linear scale for log of &lt;span style=&quot; font-weight:600;&quot;&gt;λ&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 </property>
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="0" column="1">
+                <widget class="QSlider" name="smoothnessSlider">
```

Comment:
Can we have some default values for both sliders? If there is a default value, it should also be displayed in the value label. Currently, it does not show any selected value or range for either of the sliders. 
![image](https://user-images.githubusercontent.com/28925846/49155113-24767680-f340-11e8-877a-ad8c19136b93.png)

Reply:
I am assigning these default values (same as in EIC.cpp): smoothness - 5, asymmetry - 3. Their respective labels will also have these as default values and change on any further `valueChanged` signal.

Reply:
The Asymmetry label shows '3' at the start not '0.03'
The default algorithm is AsLss at the moment. I think it might be due to QtCreator. Before saving the file, change back the tab to Threshold and the main tab to Instrumentation tab. It saves the state of the file.

Reply:
It was actually the absence of new baseline settings in "default_settings.xml" and the generated "settings.h" file.